lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,15 @@
+desc "Send Welcome Notifications"
+task broadcast_welcome_notification_flow: :environment do
+  # Should run once every hour.
+  # In order to prevent new users from receiving multiple welcome notifications in a day,
+  # a feature_live_date is required. The script will only be effective after feature_live_date
+  # and will ultimately be superseded by 7.days.ago when it's larger than feature_live_date
+  # The feature_live_date can then be removed.
```

Comment:
```suggestion
  # The feature_live_date can then be removed.
```
Hm, what about being able to turn the notification rake task on/off? I think we still want to be able to change this date pretty easily, and if we made it an ENV var, we would want to take the newly-set "live date" into account

Reply:
yeah good idea

Reply:
I've removed that comment